go/api/response_handlers.go

```diff
@@ -536,6 +537,57 @@ func handleKeyWithMemberAndScoreResponse(response *C.struct_CommandResponse) (Re
 	return CreateKeyWithMemberAndScoreResult(KeyWithMemberAndScore{key, member, score}), nil
 }
 
+func handleKeyWithArrayOfMembersAndScoresResponse(
+	response *C.struct_CommandResponse,
+) (Result[KeyWithArrayOfMembersAndScores], error) {
+	defer C.free_command_response(response)
+
+	if response == nil || response.response_type == uint32(C.Null) {
```

Comment:
```suggestion
	if response.response_type == uint32(C.Null) {
```
I think when `response` is `nil` we're in trouble. It is not the same response contains `Null` only.